src/pkgcheck/checks/eclass.py

```diff
@@ -98,18 +98,38 @@ def desc(self):
         return f"invalid pre-inherit placement, line {self.lineno}: {self.line!r}"
 
 
+class ProvidedEclassInherit(results.LineResult, results.Style):
+    """An ebuild inherits eclass which is already provided by another eclass.
+
+    When inheriting an eclass which declares ``@PROVIDES``, those eclasses
+    are guaranteed to be provided by the eclass. Therefore, inheriting them
+    in ebuild is redundant and should be removed.
+    """
+
+    skip_eclasses = ("multibuild", "multibuild-minimal")
```

Comment:
```suggestion
    skip_eclasses = ("multibuild",)
```

What's `multibuild-minimal`? xP

That said, I'm still wondering if exclusion list is the right approach. Perhaps we shouldn't be doing `@PROVIDES` for multibuild like this, and instead redocument the vars in the relevant eclasses. After all, they have kind of special values there.